mgmtsystem/ec2.py

```diff
@@ -537,3 +540,50 @@ def delete_loadbalancer(self, loadbalancer):
 
     def get_all_unused_network_interfaces(self):
         return [eni for eni in self.api.get_all_network_interfaces() if eni.status == "available"]
+
+    def import_image(self, s3bucket, s3key, format="vhd", description=None):
+        self.logger.info(" Importing image {} from {} bucket with description {} in {} started "
+                         "successfully."
+                         "format".format(s3key, s3bucket, description, format))
```

Comment:
I recommend following the logger syntax (https://docs.python.org/2/library/logging.html#logging.debug)